### ISSUE TITLE
Interview questions: vpc.md

### ISSUE THREAD

**Issue by smritidn6** [iam-veeramalla/aws-devops-zero-to-hero#45]
12. Can you peer VPCs in different regions?
No, VPC peering is limited to VPCs within the same region. To connect VPCs across regions, you would need to use VPN or AWS Direct Connect.

But AWS document says that we can peer the vpc across reions:

**Comment by ghost** [comment]
Yes, you can peer VPCs across different regions using Inter-Region VPC Peering.

Go through the decumentation here :- https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html
